src/main/java/com/actiontech/dble/services/manager/handler/DbGroupHAHandler.java

```diff
@@ -20,7 +20,7 @@ public final class DbGroupHAHandler {
 
     private static final String DB_NAME_FORMAT = "a-zA-Z_0-9\\-\\."; //nmtoken
     private static final Pattern PATTERN_DH_DISABLE = Pattern.compile("@@disable\\s*name\\s*=\\s*'([" + DB_NAME_FORMAT + "]+)'" +
-            "\\s*(instance\\s*=\\s*'([" + DB_NAME_FORMAT + "\\,]+)')?$", Pattern.CASE_INSENSITIVE);
+            "\\s*(name\\s*=\\s*'([a-zA-Z_0-9\\-]+)')?$", Pattern.CASE_INSENSITIVE);
```

Comment:
What is the meaning of changing to name here?

Reply:
get